### DISCUSSION TITLE
Games are not in FrivReborn

### DISCUSSION THREAD

**Discussion by betaadev** [Tsyuay/FrivReborn#11] [General]
Games are NOT in FrivReborn

- Super Mario Flash 
- Noitcelfer (edit: i have the [SWF file](https://cache.armorgames.com/files/games/noitcelfer-13095.swf?v=1373587527))
- Cubob (my favorite friv game)
- Friv Mobile : Love Tester
- Slice The Box
- idk

**Comment by butti2004** [comment]
also just to add a game on this list
Sea of glomp
its a pretty old game, it was probably removed because there is blood
the SWF file is stored with bluemaxima's flashpoint too

**Comment by Tsyuay** [comment]
Hello, Friv Mobile Love Tester will probably not be added because it's just a link to the modern Friv. And Super Mario Flash will also maybe not, because it was removed from Friv Classic Reborn because I was worried of a copyright issue. And Sea of Glomp also maybe not because if ti was removed from Friv, it's impossible to add, and it's also a game where there's blood, so more low probabilities.
Regards, Panda Fake Projects.
![noflash1](https://user-images.githubusercontent.com/83605971/170894339-33a5cd6e-c7f7-4dc6-aa42-823718c50c93.png)

**Reply by Danner360** [reply]
Hello, is the friv reborn project still in development? If so, is there a way to donate to work?

**Reply by Tsyuay** [reply]
I'm not being very active on it, so it's question of time to see if I can complete it myself or if I get a proffesional contributor, sadly there's nothing to donate yet.
